projects/plugins/jetpack/class.json-api-endpoints.php

```diff
@@ -1388,109 +1388,118 @@ public function get_author( $author, $show_email_and_ip = false ) {
 		$nice       = null;
 		$url        = null;
 		$ip_address = isset( $author->comment_author_IP ) ? $author->comment_author_IP : '';
+		$site_id    = -1;
 
 		if ( isset( $author->comment_author_email ) ) {
-			$id          = ( isset( $author->user_id ) && $author->user_id ) ? $author->user_id : 0;
-			$login       = '';
-			$email       = $author->comment_author_email;
-			$name        = $author->comment_author;
-			$first_name  = '';
-			$last_name   = '';
-			$url         = $author->comment_author_url;
-			$avatar_url  = $this->api->get_avatar_url( $author );
-			$profile_url = 'https://gravatar.com/' . md5( strtolower( trim( $email ) ) );
-			$nice        = '';
-			$site_id     = -1;
+			$id         = ( isset( $author->user_id ) && $author->user_id ) ? $author->user_id : 0;
```

Comment:
Since we're here, maybe we can simplify this a bit? It may be an opportunity to cast the return user ID. I don't know if that's necessary, but it may avoid any issues since we check `0 < $id` later.

```suggestion
			$id         = ! empty( $author->user_id ) ? (int) $author->user_id : 0;
```

Reply:
Thank you for the review, @jeherve.  
I didn't want to touch this old code since refactoring wasn't the main goal of my PR. But I absolutely agree — it can and should be simplified.
Fixed in c1925269ea1ad3bea419106ffc6cf4d3aa42d577